include/deal.II/base/auto_derivative_function.h

```diff
@@ -178,9 +181,10 @@ class AutoDerivativeFunction : public Function<dim>
    * Compute numerical difference quotients using the preset
    * #DifferenceFormula.
    */
-  virtual void gradient_list (const std::vector<Point<dim> > &points,
-                              std::vector<Tensor<1,dim> >    &gradients,
-                              const unsigned int              component = 0) const override;
+  virtual void
+  gradient_list(const std::vector<Point<dim>> &points,
+                std::vector<Tensor<1, dim>> &  gradients,
```

Comment:
Same, but more obvious

Reply:
https://github.com/llvm-mirror/clang/blob/release_39/lib/Format/WhitespaceManager.cpp#L327-L340

Reply:
I am awestruck that within *five minutes* you could point to the actual code in clang-format (and associated fixme) that does exactly this.